app/features/login/login_controller.rb

```diff
@@ -33,8 +36,14 @@ class LoginController < ControllerBase
 
     # Login with fastlane.ci credentials
     get "#{HOME}/ci_login" do
+      client = Octokit::Client.new(access_token: FastlaneCI.env.clone_user_api_token)
```

Comment:
I just realized we're accessing Octokit in a controller, is @taquitos ok with this?